vision/emergent_object/emergent_object_processing.py

```diff
@@ -0,0 +1,161 @@
+"""Picks the most likely emergent object out of the pool of candidates"""
+
+from nptyping import NDArray, Shape, Float64
+
+import numpy as np
+import geopy.distance
+
+from vision.common.bounding_box import BoundingBox, ObjectType, Vertices
+
+from vision.common.constants import Location, ODLC_Dict
+
+
+def pick_emergent_object(humanoids: list[BoundingBox], odlcs: ODLC_Dict) -> BoundingBox:
+    """
+    Evaluates all of the potential emergent objects to pick the most likely candidate.
+    The metrics that are used in the evaluation are, in order:
+
+    Distance to ODLC Standard Object: 50% of final score
+        The judges are more likely to be close to the ODLC Standard Object. Thus, candidates
+        that are farther away will be favored.
+
+    Distance to Emergent Object (in a DIFFERENT image): 25% of final score
+        This will be smaller for the emergent object - because we have detected it in the
+        same place twice. Judges will be walking around a lot more and will be detected in
+        different places at different times.
+
+    Bounding Box Area: 20% of final score:
+        The Emergent Object will be placed on the ground and (presumably) not standing up.
+        This means that the area of its bounding box should, in general, be larger than
+        that of the judges.
+
+    AI Confidence Level: 5% of final score:
+        Higher confidence levels should be favored
```

Comment:
This would be better to put under a `Notes` section since it's somewhat lengthy: https://missourimrr.github.io/docs/python/documenting-code/docstrings/#notes-section